src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php

```diff
@@ -328,7 +329,7 @@ public function configureOptions(OptionsResolver $resolver)
             'preferred_choices' => array(),
             'group_by' => null,
             'empty_data' => $emptyData,
-            'empty_value' => $emptyValue, // deprecated
+            'empty_value' => new \Exception(), // deprecated
```

Comment:
This is not really nice, but as the empty value can be nearly everthing it was best I could think of.